terraform/deployments/cluster-services/kubescape.tf

```
@@ -0,0 +1,24 @@
+data "aws_secretsmanager_secret" "kubescape-account" {
+  name = "govuk/kubescape"
+}
+
+data "aws_secretsmanager_secret_version" "kubescape-account-guid" {
+  secret_id = data.aws_secretsmanager_secret.kubescape-account.id
+}
+
+resource "helm_release" "kubescape" {
+  chart      = "armo-cluster-components"
+  name       = "armo"
+  namespace  = "armo-system"
+  repository = "https://armosec.github.io/armo-helm/"
+  version    = "1.7.0"
+  set {
+    name  = "clusterName"
+    value = "govuk-${var.govuk_environment}"
```

Can avoid hardcoding the `govuk` like this (if you're not keen on the long name, could add a `cluster_name` in locals in main.tf and use `local.cluster_name`). Helps avoid name conflicts in the case where we want to bring up a temporary cluster for testing turnup-related stuff.
```suggestion
    value = "${data.terraform_remote_state.cluster_infrastructure.outputs.cluster_id}-${var.govuk_environment}"
```